packs/gravitational_teleport.yml

```diff
@@ -10,4 +10,5 @@ PackDefinition:
     - Teleport.SuspiciousCommands
     # Globals used in these detections
     - panther_base_helpers
+    - panther_default
```

Comment:
Is this used?

Reply:
Oops, no. I had some helpers in `panther_default`, but I moved them to `panther_base_helpers`.